src/client/diagnosticUtil.ts

```diff
@@ -138,3 +144,11 @@ const parseDefault = (component: SourceComponent, message: DeployMessage): Compo
 
   return diagnostic;
 };
+
+const getProblemFromMessage = (message: string | DeployMessage): string => {
```

Comment:
will throw on undefined problem, which the previous version did not, even though it depended it on it being a string it could call `split` on.